change/@fluentui-react-e3da4668-61a1-4a12-a488-fd618c230554.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33136&commitId=a6261b7f7dc7cb2689da4f3ba838b976c259882b&env=prod&clientType=FLUENTUI)


  <details><summary><b>Callout</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.No callout width specified.chromium.png|2319| Changed|
|Callout.Bottom Left Edge RTL.chromium.png|2178| Changed|
|Callout.Top left edge.chromium.png|2307| Changed|
|Callout.Bottom Right Edge RTL.chromium.png|1593| Changed|
|Callout.Top auto edge.chromium.png|2307| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>